os/se/konai/konai.c

```diff
@@ -50,7 +50,7 @@ int hal_set_key(_IN_ hal_key_type mode, _IN_ uint32_t key_idx, _IN_ hal_data *ke
 	return 0;
 }
 
-int hal_get_key(_IN_ uint32_t key_idx, _OUT_ hal_data *key)
+int hal_get_key(_IN_ hal_key_type mode, _IN_ uint32_t key_idx, _OUT_ hal_data *key)
```

Comment:
Konai is one of SE feature, this should be moved to drivers\se\konai

Reply:
Okay, we'll discuss and maybe change the current structure.